pkg/syncer/spec/spec_process.go

```diff
@@ -222,6 +222,9 @@ func (c *Controller) ensureSyncerFinalizer(ctx context.Context, gvr schema.Group
 }
 
 func (c *Controller) applyToDownstream(ctx context.Context, gvr schema.GroupVersionResource, downstreamNamespace string, upstreamObj *unstructured.Unstructured) error {
+	if gvr.Group == "" && gvr.Version == "v1" && gvr.Resource == "namespaces" {
```

Comment:
Let's add a comment why we're doing this

Reply:
Please give me the comment to add as I don't know that code and why we have to do that.